docs/python_package.md

```diff
@@ -56,19 +56,19 @@ objects in a single image message at a time and publish detection results on a d
 ## [`image_recognition_service.py`](../ros/src/mas_perception_libs/image_recognition_service.py)
 
 ### `RecognizeImageService`
-A wrapper for a [`RecognizeImage`](../../mcr_perception_msgs/srv/RecognizeImage.srv) service which uses
```

Comment:
probably best to remove the links here and link to the entire repository somewhere up top.